maps/Route38.asm

```diff
@@ -350,7 +350,7 @@ SchoolboyChad1BeatenText:
 	line "enough, I guess."
 	done
 
-UnknownText_0x1a200e:
+SchoolBoyChadTestHogText:
```

Comment:
"Hog" is not a word used anywhere here, and it's hard to see how it even relates to what he's saying. Also, trainers can have standard Seen/Beaten/After labels.

Reply:
If we're going with functional labels (i.e., labelling text by where it is shown), you could use something like `SchoolboyChadBeforePhoneNumberText`.

Reply:
@aaaaaa123456789 Doing that implies doing the same for everything, which would be nice, but this PR is already huge enough.